Enigmatry.Entry.AspNetCore.Tests.SystemTextJson/Http/HttpResponseMessageAssertionsExtensions.cs

```diff
@@ -13,6 +13,11 @@ namespace Enigmatry.Entry.AspNetCore.Tests.SystemTextJson.Http;
 [PublicAPI]
 public static class HttpResponseMessageAssertionsExtensions
 {
+    private static readonly JsonSerializerOptions Options = new()
+    {
+        PropertyNameCaseInsensitive = true
+    };
```

Comment:
What was the motivation to change this now? What is the default "case sensitivity" setup in an MVC app that uses SystemTextJson.

Reply:
https://learn.microsoft.com/en-us/dotnet/core/compatibility/aspnet-core/8.0/problemdetails-custom-converters

Sorry for not letting you know before, but this is the cause of broken tests on .net 8 branch for Blueprint.
Other fixes are more reusability improvements after using Search Blocks on that new project.